src/module/item/physical/helpers.ts

```diff
@@ -10,11 +10,11 @@ class CoinsPF2e implements Coins {
     gp: number;
     pp: number;
 
-    constructor(data: Coins = {}) {
-        this.cp = data.cp ?? 0;
-        this.sp = data.sp ?? 0;
-        this.gp = data.gp ?? 0;
-        this.pp = data.pp ?? 0;
+    constructor(data?: Coins | null) {
```

Comment:
I'm suspecting that was an aberrant case from a single user--does this need to be a thing the helper?

Reply:
new CoinsPF2e(null) leading to 0 coins kinda makes sense for the API to do anyways.